google-feeds.html

```diff
@@ -215,6 +230,19 @@
           this._feedsCallbacks.push(callback);
         }
       }
+    },
+
+    _format: function() {
```

Comment:
I've tried that and does not work, the computed prop. is always executed before the lib has been loaded so the computed prop. always will be the *default* or *null* until the observed property change again.